source/Ship.cpp

```diff
@@ -2280,22 +2280,32 @@ bool Ship::IsReadyToJump(bool waitingIsReady) const
 	if(!fuelCost || fuel < fuelCost)
 		return false;
 	
+	bool hasHyperdrive = attributes.Get("hyperdrive");
+	bool hasJumpDrive = attributes.Get("jump drive");
+	// Can't jump without a drive
+	if(!hasHyperdrive && !hasJumpDrive)
+		return false;
```

Comment:
This is redundant with the fuelCost check above; if the player doesn't have a jump drive or a hyperdrive, JumpFuel will return 0. This also makes the hasHyperdrive and hasJumpDrive bools unnecessary.

Reply:
I will make the correction and push an update with the fix.